base/cvd/cuttlefish/host/libs/web/caching_build_api.cpp

```diff
@@ -75,22 +75,25 @@ CachingPaths ConstructCachePaths(const std::string& cache_base,
 
 }  // namespace
 
-CachingBuildApi::CachingBuildApi(
+template<typename BaseBuildApi>
```

Comment:
Now that these functions are `template`d, can they be moved to the header so they can be instantiated in other source files?

Reply:
It's fine to postpone, can be changed later if necessary. I thought I would find something in the [style guide](https://google.github.io/styleguide/cppguide.html) on this subject, but it doesn't make any statements either way.